web/viewer.css

```diff
@@ -643,14 +641,20 @@ html[dir='rtl'] .findbar .splitToolbarButton .toolbarButton:last-child {
 }
 
 .dropdownToolbarButton {
-  width: 120px;
-  max-width: 120px;
+  min-width: 158px;
+  max-width: 158px;
```

Comment:
Increasing these values will lead to *a lot* of wasted space for locales where the relevant strings are short, please revert this change!